tests/ci/cherry_pick.py

```diff
@@ -402,11 +402,21 @@ def receive_release_prs(self):
         logging.info("Getting release PRs")
         self.release_prs = self.gh.get_release_pulls(self._repo_name)
         self.release_branches = [pr.head.ref for pr in self.release_prs]
-        self.labels_to_backport = [
-            f"v{branch}-must-backport" for branch in self.release_branches
-        ]
 
-        if self._fetch_from:
+        if not self._fetch_from:
+            self.labels_to_backport = [
+                f"v{branch}-must-backport" for branch in self.release_branches
+            ]
+        else:
+            fetch_release_prs = self.gh.get_release_pulls(self._fetch_from)
+            fetch_release_branches = [pr.head.ref for pr in fetch_release_prs]
+            self.labels_to_backport = [
+                f"v{branch}-must-backport" for branch in fetch_release_branches
+            ]
+
+            # Forcefully add also non cloud backports
+            self.labels_to_backport += [Labels.MUST_BACKPORT]
```

Comment:
This is the only thing I'd like to fix. Let's make `--must-create-backport-label` a list and pass it to `Backport.must_create_backport_labels` instead of `Backport.must_create_backport_label`.

Maybe, I'd think of the `--aggressive-backport`, but your solution is very elegant.

Reply:
Or simpler. Since we always use `Labels.MUST_BACKPORT`, we can add `--extra-create-backport-label`. Empty by default, adding `MUST_BACKPORT_CLOUD`

Reply:
Changed to a list. The simpler option would be to remove options and have just 2 modes: public and private. No need to support custom tags, repo names, etc